### PR TITLE
Feature teardown

### PR DESCRIPTION
**Pull request by Varun2323**
Description:

Project based on Smashing Magazine page, it shows a Search article button on the top-right corner, a main section with  data and related information and a footer section.
 
 Header :
 
 	- A Navbar with multiple options based on user's preference.
 	- A button for Search Article.
 	
 Main section:
 
 	- Embedded images for the logo of the site and many different URLs for further information.
 	- Latest articles with smashing membership information.
 	- A button for creating the account.
 	- Person of the week section and Design Systems section.
 	
 Footer section:
    - Smashing magazine topics related URL's
 	- URLs redirecting to Legal, Privacy, and Security sites with further information.
 	- Text related to legal information about the website.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1067 @@
...
+  display: flex;
+  margin-top: 12px;
+  margin-left: 15px;
+}
+
+.nav-item {
+  display: block;
+  list-style: none;
+  font-size: 18px;
+  font-weight: 550;
```
Comment (arslanbisharat) [inline style.css]: Please use a specific font-weight like bold or bolder etc

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1067 @@
...
+}
+
+.nav-container {
+  display: flex;
+  align-items: flex-start;
+  text-transform: capitalize;
+  padding-top: 5px;
+  padding-bottom: 25px;
+  padding-left: 15px;
+  padding-right: 15px;
```
Comment (arslanbisharat) [inline style.css]: **Require shorthand properties:**  The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1067 @@
...
+}
+
+.more-btn {
+  display: flex;
+  background: #686868;
+  border-radius: 10px;
+  padding-top: 5px;
+  padding-bottom: 5px;
+  padding-left: 10px;
+  padding-right: 10px;
```
Comment (arslanbisharat) [inline style.css]: **Require shorthand properties:**  The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1067 @@
...
+  margin-left: 5px;
+  margin-right: 40px;
+  font-size: 18px;
+  font-weight: 550;
```
Comment (arslanbisharat) [inline style.css]: Please use a specific font-weight like bold or bolder etc

File: style.css [hunk 5]
```diff
@@ -0,0 +1,1067 @@
...
+  color: gray;
+  padding: 0;
+}
+
+.searchArti-form input {
+  display: block;
+  width: 285px;
+  height: 53px;
+  border-radius: 10px;
+  padding-left: 50px;
```
Comment (arslanbisharat) [inline style.css:161]: **Optional** Using width with padding-left can sometimes make elements larger than you expect.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,1067 @@
...
+  max-height: 40px;
+  font-size: 17px;
+  font-weight: 500;
+  color: #fff;
+  border: solid 1px #fff;
+  border-radius: 10px;
+  padding-top: 10px;
+  padding-right: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
```
Comment (arslanbisharat) [inline style.css:208]: The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,1067 @@
...
+  font-size: 45px;
+  margin-top: -3px;
+  margin-right: 15px;
+  color: gray;
+  background: white;
+  border-radius: 10px;
+  padding-top: 5px;
+  padding-right: 0;
+  padding-bottom: -5px;
+  padding-left: 5px;
```
Comment (arslanbisharat) [inline style.css]: The properties padding-top, padding-bottom, padding-left, padding-right can be replaced by padding.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,1067 @@
...
+}
+
+.serious-icon {
+  transform: rotate(-20deg);
+}
+
+.serious-icon i {
+  width: 50px;
+  height: 50px;
+  border: transparent;
```
Comment (arslanbisharat) [inline style.css:223]: **Optional** Using height with border can sometimes make elements larger than you expect.

File: style.css [hunk 9]
```diff
@@ -0,0 +1,1067 @@
...
+  background: #fff;
+  color: gray;
+}
+
+.serious-icon {
+  transform: rotate(-20deg);
+}
+
+.serious-icon i {
+  width: 50px;
```
Comment (arslanbisharat) [inline style.css]: **optional** Using width with border can sometimes make elements larger than you expect.

File: style.css [hunk 10]
```diff
@@ -0,0 +1,1067 @@
...
+  height: 50px;
+  border: transparent;
+  font-size: 45px;
+  margin-top: -3px;
+  margin-right: 15px;
+  color: gray;
+  background: white;
+  border-radius: 10px;
+  padding-top: 5px;
```
Comment (arslanbisharat) [inline style.css]: **optional** Using height with padding-top can sometimes make elements larger than you expect.

File: style.css [hunk 11]
```diff
@@ -0,0 +1,1067 @@
...
+  margin-top: -50px;
+  margin-right: 40px;
+  filter: grayscale(1);
+}
+
+.last-links {
+  margin-top: 20px;
+  margin-bottom: 0;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (arslanbisharat) [inline style.css]: The properties margin-top, margin-bottom, margin-left, margin-right can be replaced by margin.

File: style.css [hunk 12]
```diff
@@ -0,0 +1,1067 @@
...
+  width: 1000px;
+  margin-left: 50px;
+}
+
+.foot-para {
+  width: 450px;
+  margin-top: 60px;
+  margin-bottom: 30px;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (arslanbisharat) [inline style.css]: The properties margin-top, margin-bottom, margin-left, margin-right can be replaced by margin.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet. You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ] Please use dry CSS.

- [ ] Please resolve all the suggestions and I have left some for you to know. This will help you to get familiar with dry CSS. This is very important in the front-end development career. 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
---------------------------------------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**